src/bubbleSort.js

```diff
@@ -0,0 +1,26 @@
+export default function bubbleSort(array){
+  var sortingHat = [];
+  if(Array.isArray(array)) {
```

Comment:
The else blocks here aren't really necessary - to do this in fewer lines, you can just have your code check for the cases where you would return the error message before continuing.

``` javascript
if (!Array.isArray(array)) {
    return 'Please enter an array of numbers';
}
for ( var i = 0; i < array.length; i++ ) {
    if (isNaN(array[i]) {
        return 'Please enter an array of numbers';
    }
    sortingHat.push(array[i])
}
```